app/assets/javascripts/components/OtherSessionsLogout.tsx

```diff
@@ -0,0 +1,78 @@
+import { useRef, useState } from 'preact/hooks';
+import {
+  AlertDialog,
+  AlertDialogDescription,
+  AlertDialogLabel,
+} from '@reach/alert-dialog';
+import { WebApplication } from '@/ui_models/application';
+import { AppState } from '@/ui_models/app_state';
+import { observer } from 'mobx-react-lite';
+import { FunctionComponent } from 'preact';
+
+type Props = {
+  application: WebApplication;
+  appState: AppState;
+};
+
+export const OtherSessionsLogoutContainer = observer((props: Props) => {
+  if (!props.appState.accountMenu.otherSessionsLogOut) {
+    return null;
+  }
+  return <ConfirmOtherSessionsLogout {...props} />;
+});
+
+const ConfirmOtherSessionsLogout = observer(
+  ({ application, appState }: Props) => {
+
+    const cancelRef = useRef<HTMLButtonElement>();
+    function closeDialog() {
+      appState.accountMenu.setOtherSessionsLogout(false);
+    }
+
+    return <AlertDialog onDismiss={closeDialog} leastDestructiveRef={cancelRef}>
```

Comment:
Could we please add a newline here after return so it's easier to read?

Reply:
Done.